examples/smartContracts/index.ts

```diff
@@ -208,8 +210,8 @@ const pollAsyncEvents = async (
     ).start();
     const args = new Args();
     const result = await web3Client.smartContracts().readSmartContract({
-      fee: 0,
-      maxGas: 700000,
+      fee: 0n,
+      maxGas: BigInt(700000),
```

Comment:
why not 700_000n? same line 233
```suggestion
      maxGas: 700_000n,
```

Reply:
changed, thanks.